Kosystem/wwwroot/css/site.css

```diff
@@ -0,0 +1,29 @@
+html, body {
+    height: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=5636989)